JS/Clue.js

```diff
@@ -1,13 +1,52 @@
 class Clue {
+  constructor() {
+    this.question;
```

Comment:
This should probably still have some sort of default value. What will the data type be eventually? What would be a good default value for that data type?